Automattic/sensei
# Renamed learner -> student in all user-facing occurrences

Fixes #4451
Trello: https://trello.com/c/NQx3rfMI/17-replace-all-instances-of-learner-with-student

## Changes proposed in this Pull Request

- Replaced learner -> student in all of the strings that are visible to the user
## Why I didn't change it in the code: 
- If it is going to be done it needs to be done in separate PR, not to make this one 2complicated 
- Public functions and block names CAN't be renamed without some bigger changes
- It will leave the codebase in half-state

## How I did it 
- I used .pot file to find all the user-facing occurrences
- Pot was easy to search and also includes files + lines where this occurrence happened
- After I replaced occurrences in the code I re-installed it and regenerated pot file

## Testing instructions
1. Go to the pot file -> see there aren't any msg that include **learner** **learners**
2. Start wp-admin with sensei plugin locally and go trough setting, see if there is any **learner**  left
